imap_processing/cdf/config/imap_ultra_global_cdf_attrs.yaml

```diff
@@ -0,0 +1,151 @@
+
+imap_ultra_l1b_45sensor-cullingmask:
+    Descriptor: 45sensor-cullingmask
+    TEXT: *txt
+    Data_level: 1B
+    Data_type: L1B_Culling-Mask>Level-1B Culling Mask
+    Logical_source: imap_ultra_l1b_45sensor-cullingmask
+    Logical_source_description: IMAP-Ultra Instrument Level-1B Culling Mask Data.
+
+imap_ultra_l1b_90sensor-cullingmask:
+    Descriptor: 90sensor-cullingmask
+    TEXT: *txt
+    Data_level: 1B
+    Data_type: L1B_Culling-Mask>Level-1B Culling Mask
+    Logical_source: imap_ultra_l1b_90sensor-cullingmask
+    Logical_source_description: IMAP-Ultra Instrument Level-1B Culling Mask Data.
+
+imap_ultra_l1b_45sensor-badtimes:
+    Descriptor: 45sensor-badtimes
+    TEXT: *txt
+    Data_level: 1B
+    Data_type: L1B_Badtimes>Level-1B Badtimes
+    Logical_source: imap_ultra_l1b_45sensor-badtimes
+    Logical_source_description: IMAP-Ultra Instrument Level-1B Badtimes Data.
+
+imap_ultra_l1b_90sensor-badtimes:
+    Descriptor: 90sensor-badtimes
+    TEXT: *txt
+    Data_level: 1B
+    Data_type: L1B_Badtimes>Level-1B Badtimes
+    Logical_source: imap_ultra_l1b_90sensor-badtimes
+    Logical_source_description: IMAP-Ultra Instrument Level-1B Badtimes Data.
+
+imap_ultra_l1c_45sensor-pset-exposure:
+    Descriptor: 45sensor-pset-exposure
+    TEXT: *txt
+    Data_level: 1C
+    Data_type: L1C_Exposure>Level-1C Exposure
+    Logical_source: imap_ultra_l1c_45sensor-pset-exposure
+    Logical_source_description: IMAP-Ultra Instrument Level-1C Pointing Set Grid Exposure Data.
+
+imap_ultra_l1c_90sensor-pset-exposure:
+    Descriptor: 90sensor-pset-exposure
+    TEXT: *txt
+    Data_level: 1C
+    Data_type: L1C_Exposure>Level-1C Exposure
+    Logical_source: imap_ultra_l1c_90sensor-pset-exposure
+    Logical_source_description: IMAP-Ultra Instrument Level-1C Pointing Set Grid Exposure Data.
+
+imap_ultra_l1c_45sensor-pset-sensitivity:
+    Descriptor: 45sensor-pset-sensitivity
+    TEXT: *txt
+    Data_level: 1C
+    Data_type: L1C_Sensitivity>Level-1C Sensitivity
+    Logical_source: imap_ultra_l1c_45sensor-pset-sensitivity
+    Logical_source_description: IMAP-Ultra Instrument Level-1C Pointing Set Grid Sensitivity Data.
+
+imap_ultra_l1c_90sensor-pset-sensitivity:
+    Descriptor: 90sensor-pset-sensitivity
+    TEXT: *txt
+    Data_level: 1C
+    Data_type: L1C_Sensitivity>Level-1C Sensitivity
+    Logical_source: imap_ultra_l1c_90sensor-pset-sensitivity
+    Logical_source_description: IMAP-Ultra Instrument Level-1C Pointing Set Grid Sensitivity Data.
+
+imap_ultra_l1c_45sensor-pset-counts:
+    Descriptor: 45sensor-pset-counts
+    TEXT: *txt
+    Data_level: 1C
+    Data_type: L1C_Sensitivity>Level-1C Counts
+    Logical_source: imap_ultra_l1c_45sensor-pset-counts
+    Logical_source_description: IMAP-Ultra Instrument Level-1C Pointing Set Grid Counts Data.
+
+imap_ultra_l1c_90sensor-pset-counts:
+    Descriptor: 90sensor-pset-counts
+    TEXT: *txt
+    Data_level: 1C
+    Data_type: L1C_Sensitivity>Level-1C Counts
```

Comment:
I also think this data_type might need to include a 90/45 distinction. I also think they will need to be distinct? I don't remember what SPDF told me about it, but I think they need to be very specific.